tests/set-annotation-empty-list/fn-config.yaml

```diff
@@ -7,5 +7,4 @@ metadata:
       container:
         image: gcr.io/kpt-fn/set-annotation:unstable
     config.kubernetes.io/local-config: 'true'
-data:
-  annotations: []
+data: {}
```

Comment:
Can you please add a similar test for using CRD?

Same comment for set-label and set-namespace